static/app/views/partnershipAgreement/index.tsx

```diff
@@ -0,0 +1,36 @@
+import {Label} from 'sentry/components/editableText';
+import Form from 'sentry/components/forms/form';
+import NarrowLayout from 'sentry/components/narrowLayout';
+import {t, tct} from 'sentry/locale';
+
+export type ParntershipAgreementType = 'standard' | 'partner_presence';
+
+type Props = {
+  agreements: Array<ParntershipAgreementType>,
+  partnerDisplayName: string,
+  onSubmitSuccess?: () => void;
+};
+
+export default function PartnershipAgreement({partnerDisplayName, agreements, onSubmitSuccess}: Props) {
+  const standardAgreement = tct(
+    'This organization is created in partnership with [partnerName]. By pressing continue, you acknowledge that you have agreed to Sentry’s [tosLink] and [ppLink] through the partner’s application[endOfSentence]',
+     {
+      partnerName: partnerDisplayName,
+      tosLink: <a href='https://sentry.io/terms/' target='blank'>terms of service</a>,
```

Comment:
In order to also translate the link text, you can do `[tosLink:terms of service]` in the text, then `<ExternalLink href="..." />` here